src/web/mjs/connectors/PlotTwistNoFansub.mjs

```diff
@@ -11,8 +9,47 @@ export default class PlotTwistNoFansub extends WordPressClarityMangaReader {
         this.links = {
             login: 'https://www.plot-twistnf-scans.com/wp-login.php'
         };
-
         this.paths = [ '/proyectos-finalizados-new/', '/proyectos-activosss/' ];
         this.queryMangas = 'div.vc_gitem-zone a.vc_gitem-link';
     }
-}
\ No newline at end of file
+    async _getChapterList( manga, callback ) {
+        let uri = new URL( manga.id, this.url );
+        let request = new Request(uri.href, this.requestOptions );
```

Comment:
strange mix of async & callback 
it should be async _getChapter (instead of list) and then just return

Reply:
Its because the template is still old school code. Not sure something wont go wrong if i use new style.
Well, gotta rewrite it

Reply:
new style is just removing callback, add async in front, remove List from the function name  and using a return :)